src/Controller/EnMarche/CommitteeController.php

```diff
@@ -108,6 +108,13 @@ public function unfollowAction(Request $request, Committee $committee): Response
             throw $this->createAccessDeniedException('Invalid CSRF protection token to unfollow committee.');
         }
 
+        if ($this->isGranted(CommitteePermissions::SUPERVISE, $committee)
+            || ($this->isGranted(CommitteePermissions::HOST, $committee)
+                && !$this->get('app.committee.manager')->hasSeveralHostsAndSupervisors($committee))) {
```

Comment:
Must be done in the voter to make the logic reusable in controllers and twig templates.